aspnet/web-forms/overview/data-access/editing-inserting-and-deleting-data/limiting-data-modification-functionality-based-on-the-user-vb.md

```diff
@@ -34,7 +34,7 @@ In this tutorial we'll examine how to dynamically adjust the data modification c
 Let s get started!
 
 > [!NOTE]
-> ASP.NET 2.0 s membership system provides a standardized, extensible platform for creating, managing, and validating user accounts. Since an examination of the membership system is beyond the scope of these tutorials, this tutorial instead "fakes" membership by allowing anonymous visitors to choose whether they are from a particular supplier or from our company. For more on membership, refer to my [Examining ASP.NET 2.0 s Membership, Roles, and Profile](http://aspnet.4guysfromrolla.com/articles/120705-1.aspx) article series.
+> ASP.NET 2.0 s membership system provides a standardized, extensible platform for creating, managing, and validating user accounts. Since an examination of the membership system is beyond the scope of these tutorials, this tutorial instead "fakes" membership by allowing anonymous visitors to choose whether they are from a particular supplier or from our company. For more on membership, refer to my [Examining ASP.NET 2.0 s Membership, Roles, and Profile]`(http://aspnet.4guysfromrolla.com/articles/120705-1.aspx)` article series.
```

Comment:
```suggestion
> ASP.NET 2.0 s membership system provides a standardized, extensible platform for creating, managing, and validating user accounts. Since an examination of the membership system is beyond the scope of these tutorials, this tutorial instead "fakes" membership by allowing anonymous visitors to choose whether they are from a particular supplier or from our company. For more on membership, refer to my "Examining ASP.NET 2.0 s Membership, Roles, and Profile" article series (`http://aspnet.4guysfromrolla.com/articles/120705-1.aspx`).
```
At a minimum, we need to remove the square brackets that are used to define the link. I moved the backticks inside the parentheses (which are now text rather than link coding), and then I moved the parenthetical to the end to improve the readability. The latter is more editorial, but the bracket and backtick changes are key.

Reply:
fixed.